src/test/java/apoc/export/csv/ExportCsvTest.java

```diff
@@ -203,7 +204,7 @@ public void testExportGraphCsvWithoutQuotes() throws Exception {
     @Test
     public void testExportQueryCsv() throws Exception {
         File output = new File(directory, "query.csv");
-        String query = "MATCH (u:User) return u.age, u.name, u.male, u.kids, labels(u)";
+        String query = "MATCH (u:User) return labels(u), u.age, u.kids, u.male, u.name";
```

Comment:
the columns shouldn't be in sort order for this test otherwise we don't know it actually keeps the query return order.